clients/admin-ui/src/features/privacy-requests/privacy-requests.slice.ts

```diff
@@ -353,6 +353,63 @@ export const privacyRequestApi = createApi({
       }),
       invalidatesTags: ["Notification"],
     }),
+    getStorageDetails: build.query<any, any>({
```

Comment:
@ssangervasi - you mentioned in my previous PR that the models are auto-generated by openapi?

Reply:
Yep: https://github.com/ethyca/fides/blob/main/clients/admin-ui/src/types/api/README.md#L19-L20